app/controllers/cloud_volume_controller.rb

```diff
@@ -693,6 +693,7 @@ def form_params_create
       # Only set IOPS if io1 (provisioned IOPS) and IOPS available
       options[:iops] = params[:aws_iops] if options[:volume_type] == 'io1' && params[:aws_iops]
       options[:availability_zone] = params[:aws_availability_zone_id] if params[:aws_availability_zone_id]
+      options[:snapshot_id] = params[:aws_cloud_volume_snapshot_id] if params[:aws_cloud_volume_snapshot_id]
```

Comment:
Now that we are using `base_snapshot` properly from the API, I suggest you rename this to `aws_base_snapshot`